system-int-test/src/cmr/system_int_test/utils/url_helper.clj

```diff
@@ -428,6 +428,20 @@
   []
   (format "http://localhost:%s/bulk_index/system_concepts" (transmit-config/bootstrap-port)))
 
+(defn fingerprint-url
+  ([]
+   (format "http://localhost:%s/fingerprint/variables" (transmit-config/bootstrap-port)))
+  ([concept-id]
+   (format "%s/%s" (fingerprint-url) concept-id)))
+
+(defn fingerprint-by-provider-url
+  [provider-id]
+  (format "%s?provider=%s&synchronous=true" (fingerprint-url) provider-id))
```

Comment:
These are meant to be URLs with no query parameters. We should be passing in query parameters when we make a request.

Reply:
Done.